administrator-manual/en/docker.rst

```diff
@@ -135,6 +136,9 @@ Aeria uses a docker plugin. To update the plugin ::
 
  signal-event nethserver-docker-plugin-update
 
+
+The aeria network is not standard on docker, the developer can be contacted at https://github.com/devplayer0/docker-net-dhcp
+
```

Comment:
I'm not still convinced that this should go on the manual. It's an implementation detail, let's move to the README of the package.
```suggestion
```

Reply:
removed of the official documentation and added to the readme : https://github.com/NethServer/nethserver-docker/pull/19